src/server/pps/cmds/cmds_test.go

```diff
@@ -71,7 +71,7 @@ const (
 	secret   = "secret"
 
 	create  = "create"
-	delete  = "delete"
+	del     = "delete"
```

Comment:
This conflicted with my desire to use `builtin.delete` to delete something from a map ;)